.github/CONTRIBUTING.md

```diff
@@ -196,17 +200,30 @@ TOTAL                             1586      0   100%
 
 Certain statements can be omitted from the coverage report by adding `# pragma: no cover` but this should be used conservatively. If your tests pass and your coverage is at 100%, you're ready to [submit a pull request](https://github.com/ucfopen/canvasapi/pulls)!
 
-#### Making a Pull Request
+#### Making a pull request
 
 Be sure to include the issue number in the title with a pound sign in front of it (#123) so we know which issue the code is addressing. Point the branch at `develop` and then submit it for review.
 
-## Code Style Guidelines
+## Code style guidelines
 
 We try to adhere to Python's [PEP 8](https://www.python.org/dev/peps/pep-0008/) specification as much as possible. In short, that means:
 
 * We use four spaces for indentation.
 * Lines should be around 80 characters long, but up to 99 is allowed. Once you get into the 85+ territory, consider breaking your code into separate lines.
 
+### Running code style checks
+
+The following tools can help you check your code for style correctness. We run these tools in our CI pipeline, so running them locally is a great way to speed up acceptance of your pull requests.
+
+You can use [pre-commit](https://pre-commit.com/) to force each check to run before you create a commit locally:
```

Comment:
I think we only have pre-commit set up to run `black` at this time

Reply:
Created #433 to track this.